src/classes/ADDR_OtherAcc_TEST.cls

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.